src/components/Hero/index.tsx

```diff
@@ -29,29 +33,38 @@ const Hero: React.FC<Props> = ({
         backgroundImage: `url(${backgroundSrc})`
       }}
       className={classnames(
-        'min-h-screen bg-no-repeat bg-cover bg-center flex flex-col justify-center p-screen-spacing lg:px-desktop-screen-spacing',
+        'min-h-screen bg-no-repeat bg-cover bg-center flex items-center justify-between flex-wrap p-screen-spacing lg:px-desktop-screen-spacing',
         className
       )}
     >
-      <h1 className="text-5xl leading-13 font-black text-white mb-6 lg:leading-15 lg:text-6xl">
-        {title}
-      </h1>
-      <p className="font-bold text-2xl text-white leading-6 mb-12">{text}</p>
-      <div className="flex flex-col md:block">
-        <Button
-          className="w-full mb-6 md:w-1/3 lg:w-1/5 md:mr-6 lg:mb-0"
-          onClick={primaryAction}
-        >
-          {primaryActionLabel}
-        </Button>
-        <Button
-          className="w-full md:w-1/3 lg:w-1/5"
-          onClick={secondaryAction}
-          variant="secondary"
-        >
-          {secondaryActionLabel}
-        </Button>
+      <div>
+        <h1 className="text-4xl leading-10 font-black text-white mb-6 lg:leading-15 lg:text-6xl">
+          {title}
+        </h1>
+        <p className="font-bold text-2xl text-white leading-6">{text}</p>
+        {hasActions && (
+          <div className="flex flex-col mt-12 md:block">
+            {primaryAction && primaryActionLabel && (
```

Comment:
Maybe those props should not be optional but required to build our hero, isn't?

Reply:
On the first iteration, they were required, however, since the union page does not have any actions I needed to make them optional